### ISSUE TITLE
Add a feature to copy a portion of text from the terminal output

### ISSUE THREAD

**Issue by brizaldi** [stateful/vscode-runme#1202]
It would be very useful if we could copy partial content from the terminal output.

Currently, we can only copy using the `Copy` button provided by the cell UI, but that button will copy all the text.
![image](https://github.com/stateful/vscode-runme/assets/41714881/af77e319-c051-4aee-bf29-cfcfddf0d2d0)

**Comment by sourishkrout** [comment]
Hi @brizaldi! Thanks for filing this issue. So the idea is to "respect" the selection inside the terminal for the copy button?

**Comment by brizaldi** [comment]
@sourishkrout Yes, but it doesn't have to be through the copy button. It should also be possible to copy using the Ctrl+C shortcut or by behaving like a regular terminal where we can copy by just using right-click.
